quickstarts/hello-rules-multi-module/README.md

```diff
@@ -1,10 +1,14 @@
 ## Red Hat Decision Manager Kie Server Quickstart - Multi module
 
-This quickstart is intend to be used with the [RHDM Kie Server](https://github.com/jboss-container-images/rhdm-7-openshift-image/tree/master/kieserver) image.
+This quickstart is intend to be used with the [RHDM Kie Server](https://github.com/jboss-container-images/rhdm-7-openshift-image/tree/7.8.1.GA/kieserver) image.
 
 ## How to use it?
 
+<<<<<<< HEAD
 To deploy the Hello Rules demo you can use the [rhdm79-prod-immutable-kieserver](https://github.com/jboss-container-images/rhdm-7-openshift-image/blob/master/templates/rhdm79-prod-immutable-kieserver.yaml)
+=======
+To deploy the Hello Rules demo you can use the [rhdm78-prod-immutable-kieserver](https://github.com/jboss-container-images/rhdm-7-openshift-image/blob/7.8.1.GA/templates/rhdm78-prod-immutable-kieserver.yaml)
+>>>>>>> dadfff5... [KIECLOUD-427] - Update RHPAM and RHDM repositories for 7.8.1.GA release
```

Comment:
This looks like some unresolved conflict in cherry picking commits.

Reply:
I forgot to fix the conflicts in this readme file :)